domains/rohanbharti.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "optimisticlucifer",
+    "email": "bharti.rohaa@gmail.com"
+  },
+  "record": {
+    "CNAME": "optimisticlucifer.github.io/rohanbharti.github.io/"
```

```suggestion
    "CNAME": "optimisticlucifer.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).